src/scripts/main.js

```diff
@@ -1,3 +1,84 @@
 'use strict';
 
-// write code here
+const buttons = document.querySelector('.container');
+const table = document.querySelector('.field');
+
+buttons.addEventListener('click', (e) => {
+  const button = e.target.closest('.button');
+  const rows = table.rows;
+
+  if (button.classList.contains('append-row')) {
```

Comment:
Ensure that `button` is not null before calling `classList.contains`. This will prevent potential runtime errors.